public/app/features/provisioning/RepositoryListPage.tsx

```diff
@@ -118,7 +131,7 @@ function RepositoryListPageContent({ items }: { items?: Repository[] }) {
                   <LinkButton href={`${PROVISIONING_URL}/${name}/edit`} variant="secondary">
                     Edit
                   </LinkButton>
-                  {healthy && <SyncRepository repository={item} />}
```

Comment:
rather than hide the sync button, it will be disabled when the repository is not healthy